shotgun_api3/shotgun.py

```diff
@@ -308,9 +327,11 @@ def __init__(self,
         self.config.script_name = script_name
         self.config.user_login = login
         self.config.user_password = password
+        self.config.session_token = session_token
         self.config.sudo_as_login = sudo_as_login
         self.config.convert_datetimes_to_utc = convert_datetimes_to_utc
         self.config.no_ssl_validation = NO_SSL_VALIDATION
+        self.config.raw_http_proxy = http_proxy
```

Comment:
This could use a comment as to how it could be used.  Since it isn't used in the API itself it isn't obvious.